homeyess/website/views.py

```diff
@@ -210,6 +210,31 @@ def get_object(self):
 		id_ = self.kwargs.get('post_id')
 		return get_object_or_404(RideRequestPost, id=id_)
 
+def deletejob(request, user_id, job_id):
```

Comment:
We dont need a page for the deletion. Just delete the object then redirect to dashboard